src/main/templates/partials/header.hbs

```diff
@@ -0,0 +1,38 @@
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="mobile.css">
+  <title>ISDCF {{listTitle}}</title>
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" crossorigin="anonymous"></script>
+  <script type="application/javascript" src="features.js"></script>
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-md navbar-light fixed-top bg-light d-print-none d-flex">
+    <a class="navbar-brand" href="#">ISDCF {{listTitle}}</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a id="json-feed" target="_blank" class="nav-link" href="https://registry.isdcf.com/{{listType}}">JSON Feed</a>
+        </li>
+        <li class="nav-item">
+          <a id="pdf-download" target="_blank" class="nav-link" title="Download as PDF" href="{{pdf_path}}">Download PDF</a>
+        </li>
+      </ul>
+        <form id="search-form" class="form-inline">
+          <span class="align-middle"><input id="search" class="form-control mb-2" type="text" placeholder="Search {{listTitle}}">
+          <input type="button" class="btn btn-secondary btn-sm mb-2 clear-filter" value="x"/></span>
```

Comment:
I am really not in love with trying to come up with our own _reset_ button, i.e. additional maintenance, impact on responsiveness and accessibility, etc.

Reply:
The need is based on the fact that our "search" is actually a JQuery filter. You cannot just "clear" the field, as it doesn't reset the filtering. The same reset is also enabled when clicking on links such as obsoleted links. Prior to having the reset, if you had items filtered on the "search" and you clicked a link, it just sat there (as the filter was still on). Now it will clear the filter and then go to the proper link. The responsiveness works well on all platforms and mobile, not see any issues yet, and I'm using that same method with the doc registry which has 1000's more objects. 

Example - on the current master branch, do a filter search for Access IT, then click the link for CIN. It doesn't do anything. The new branch work properly.

Reply:
> You cannot just "clear" the field, as it doesn't reset the filtering.

This is only a problem on UAs that add an `x` in the search box, right? For those browsers, it should be possible to hook to the [search event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/search_event).

If we are going to use a custom button, then:

- we should use a [group](https://getbootstrap.com/docs/4.0/components/input-group/)
- use something other than the letter `x`

>The responsiveness works well on all platforms and mobile,

The `x` appears on a separate line.

![image](https://user-images.githubusercontent.com/4871350/88434728-a89f2c80-cdb5-11ea-8579-dc7feeaeea46.png)

Reply:
It's not just for those UAs. As the hook on the search event only works as you type or clear (one letter at a time). If you clear the field all at once, it don't clear the filter. From a UX side, clicking in and backspacing vs clear button, button wins.

And thought I put it in a group, obv that was a prior version, re-adding that in now.

Reply:
I must be missing something here. Both Safari and Chrome show a built-in "×" (and note that's not an "x") within the search box whenever it is non-empty. Clicking that "×" *or* typing `ESC` *or* `CMD-<DELETE>` will clear the search box and update the page. Isn't that pretty much all we need?

Is the only new desired functionality for the search box to clear itself when you click on a local anchor?

Reply:
This is now fixed in newest commit. Swapped field to search (clear in field) and removed reset button, changed search function to "input" vs "keyup".